tests/globalkilltest/global_kill_test.go

```diff
@@ -69,38 +69,58 @@ var _ = Suite(&TestGlobalKillSuite{})
 type TestGlobalKillSuite struct {
 	pdCli *clientv3.Client
 	pdErr error
+
+	clusterId string
+	pdProc    *exec.Cmd
+	tikvProc  *exec.Cmd
 }
 
 func (s *TestGlobalKillSuite) SetUpSuite(c *C) {
 	err := logutil.InitLogger(&logutil.LogConfig{Config: zaplog.Config{Level: *logLevel}})
 	c.Assert(err, IsNil)
 
+	s.clusterId = strings.ReplaceAll(time.Now().String(), " ", "-")
```

Comment:
Use time.Now().Format(time.RFC3339Nano) directly? So we don't need to create another time layout...:)